lib/runcible/models/python_importer.rb

```diff
@@ -0,0 +1,24 @@
+module Runcible
+  module Models
+    class PythonImporter < Importer
+      ID = 'python_importer'
+      REPO_TYPE = 'python-repo'
+
+      attr_accessor 'feed', 'max_speed', 'max_downloads', 'upstream_name',
+                    'proxy_port', 'proxy_password', 'proxy_username', 'mask_id',
+                    'packages_names'
+
```

Comment:
Anything the default importer supports, the Python type supports -- I'm trying to find again the default configs in their documentation to check this.